module/config.mjs

```diff
@@ -1787,6 +1787,36 @@ DND5E.hitDieTypes = ["d4", "d6", "d8", "d10", "d12"];
 
 /* -------------------------------------------- */
 
+/**
+ * Configuration data for rest types.
+ *
+ * @typedef {object} RestConfiguration
+ * @property {Record<string, number>} duration  Duration of different rest variants in minutes.
+ */
+
+/**
+ * Types of rests.
+ * @type {RestConfiguration}
```

Comment:
Should this be `@enum` or `Record<string, RestConfiguration>`? It doesn't look quite right to me.